.circleci/config.yml

```diff
@@ -42,18 +42,21 @@ commands:
   bootstrap_node:
     steps:
       - restore_cache:
-          name: Restore Cache - yarn
+          name: Restore Cache - pnpm
           keys:
-            - v2-dependencies-{{ checksum "package.json" }}
+            - pnpm-v1-dependencies-{{ checksum "package.json" }}
```

Comment:
If you add `&pnpm-cache-key` between the- and pnpm-... you can specify the name once and use it on L59, by doing `key: *pnpm-cache-key`.  This is done in the coda config file if you want examples.

Reply:
That's definitely better. Done